### PR TITLE
Add Postgres to dev environment

### PR DESCRIPTION
**Pull request by dougneal**
Summary
---------

Switch the devcontainer config to use docker-compose to bring up a Postgres container in addition to the Flask app container.

Test Plan
---------

Rebuild dev container environment in VSCode

### CODE REVIEW

File: open-media-match/.devcontainer/omm_config.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Database configuration
+DBUSER = 'media_match'
+DBPASS = 'hunter2'
```
Comment (Dcallies) [inline open-media-match/.devcontainer/omm_config.py:3]: lul

File: open-media-match/.devcontainer/omm_config.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# Database configuration
```
Comment (Dcallies) [inline open-media-match/.devcontainer/omm_config.py:1]: I'm inherently suspicious of turing complete configuration files, but I think this is fine in the short term. 

I've read enough about YAML to be scared, and tend towards just JSON, but this is all irrelevant in first pass.

File: open-media-match/src/OpenMediaMatch/__init__.py [hunk 3]
```diff
@@ -1,13 +1,15 @@
...
 import flask
 
 
 def create_app():
     """
     Create and configure the Flask app
     """
-    app = flask.Flask(__name__, instance_relative_config=True)
+    app = flask.Flask(__name__)
+    app.config.from_envvar('OMM_CONFIG')
```
Comment (Dcallies) [inline open-media-match/src/OpenMediaMatch/__init__.py:12]: (ignorable): I think having this be a harcoded path at this stage is also fine, one less indirection to follow

**Review by Dcallies — APPROVED** [review]
This all looks fine, some popcorn gallery comments that are ignorable. Up to you to merge as is.

File: open-media-match/.devcontainer/omm_config.py (hunk 2 above)
Comment (dougneal) [reply open-media-match/.devcontainer/omm_config.py:1]: Fair point, and Flask provides a few methods by which to load config.

**Pull request merged**